python/meeting-scheduling/src/meeting_scheduling/demo_data.py

```diff
@@ -0,0 +1,221 @@
+import random
+from datetime import datetime, timedelta
+from enum import Enum
+from typing import List, Tuple, Any, Callable
+
+from .domain import Person, TimeGrain, Room, Meeting, MeetingAssignment, MeetingSchedule
+
+class DemoData(str, Enum):
+    SMALL = "SMALL"
+    MEDIUM = "MEDIUM"
+    LARGE = "LARGE"
+
+
+def generate_demo_data() -> MeetingSchedule:
+    """Generate demo data for the meeting scheduling problem."""
+    random.seed(0)  # For reproducible results
```

Comment:
Please create an instance of Random and pass it where required.
```suggestion
from random import Random
from datetime import datetime, timedelta
from enum import Enum
from typing import List, Tuple, Any, Callable

from .domain import Person, TimeGrain, Room, Meeting, MeetingAssignment, MeetingSchedule

class DemoData(str, Enum):
    SMALL = "SMALL"
    MEDIUM = "MEDIUM"
    LARGE = "LARGE"


def generate_demo_data() -> MeetingSchedule:
    """Generate demo data for the meeting scheduling problem."""
    random = Random(0)  # For reproducible results
```